src/quantifiers/index.ts

```diff
@@ -36,9 +37,20 @@ export function zeroOrMore(...children: RegexElement[]): ZeroOrMore {
   };
 }
 
+export function repeat(
+  config: Repeat['config'],
```

Comment:
[nit] Pls extract `RepeatConfig` or `RepeatProps` type to avoid using `['key']` syntax